app/views/static_pages/home.html.erb

```diff
@@ -1,74 +1,70 @@
-        <li><a href="<%= login_path %>" class="button secondary" rel="nofollow">Log In</a></li>
-        <li><a href="<%= signup_path %>" class="button primary" rel="nofollow">Sign Up</a></li>
-      <% end %>
-    </ul>
-  </nav>
-</header>
 <main id="homepage">
   <section id="plans">
     <h1>Every membership brings us closer to our goals.</h1>
     <h4>Your monthly contribution helps us plan for the future.</h4>
     <p id="notice"><%= notice %></p>
+
     <div class="subscription--plans">
-      <% @plans.each do |plan| %>
-        <%= link_to  @current_user ? new_subscription_charge_path(plan_id: plan) : login_path, class: "subscription--plan", id: "subscription-#{plan.name.parameterize.underscore}" do %>
-          <div>
-            <h3>
-              <%= plan.name %>
-              <span class="subscription--plan-price"><%= number_to_currency(plan.amount).gsub(/\.00$/, "") %><span>/mo</span></span>
-            </h3>
-            <h4><%= plan.headline %></h4>
-            <%= plan.description %>
-            <% if @current_user_subscription&.plan&.id == plan.id %>
-              <p><strong>Current subscription</strong></p>
-            <% end %>
-          </div>
+      <% @plans.each do |plan| %> <%= link_to @current_user ?
+      new_subscription_charge_path(plan_id: plan) : login_path, class:
+      "subscription--plan", id:
+      "subscription-#{plan.name.parameterize.underscore}" do %>
+      <div>
+        <h3>
+          <%= plan.name %>
+          <span class="subscription--plan-price"
+            ><%= number_to_currency(plan.amount).gsub(/\.00$/, "") %><span
+              >/mo</span
+            ></span
+          >
+        </h3>
+        <h4><%= plan.headline %></h4>
+        <%= plan.description %> <% if @current_user_subscription&.plan&.id ==
+        plan.id %>
+        <p><strong>Current subscription</strong></p>
         <% end %>
-      <% end %>
-      <%= link_to new_charge_path, class: "subscription--plan", id: "one-time-donation" do %>
-        <h3>Other</h3>
-        <h4>Pay what you can. Every dollar counts.</h4>
-        <p>“To each according to their ability, to each according to their need.” Contributing what you can, where you can is what we’re all about.</p>
-        <p>Folks who donate $100 on a one-time basis will receive a free gift from our store. People who pay dues at the rate of $100 per month or more will receive a thank you call from us.</p>
+      </div>
+      <% end %> <% end %> <%= link_to new_charge_path, class:
+      "subscription--plan", id: "one-time-donation" do %>
+      <h3>Other</h3>
+      <h4>Pay what you can. Every dollar counts.</h4>
+      <p>
+        “To each according to their ability, to each according to their need.”
+        Contributing what you can, where you can is what we’re all about.
+      </p>
+      <p>
+        Folks who donate $100 on a one-time basis will receive a free gift from
+        our store. People who pay dues at the rate of $100 per month or more
```

Comment:
That wording may need a ticket in order to think about it:

> Folks who donate $100 on a one-time basis will receive a free gift from our store. People who pay dues at the rate of $100 per month or more will receive a thank you call from us.

For sure is the truth but highlight in a very explicit way the difference and feels like undervaluing folks with dues of rate that $100 per month.

Besides, how come that someone with subscription of 100$ doesn't get a gift and a person with ONE time donation does?

An alternative will be something like:

> Receive a personal call from us to thank you about you support by joining into the movement with 100$ subscription or get a special gift from our store after one time donation of 100$ or more.

Reply:
I won't change copy on this PR, this will come on the redesign